src/moonstone-samples/lib/TimePickerSample.js

```diff
@@ -83,7 +84,7 @@ module.exports = kind({
 	},
 	timeChanged: function (sender, event) {
 		if (this.$.result && event.value){
-			var timeArray = event.value.toTimeString().split(': ');
+			var timeArray = event.value.toTimeString().split(':');
 			this.$.result.setContent(event.name + ' changed to ' + timeArray[0] + ': ' + timeArray[1]);
```

Comment:
Alternatively, if we want to display just the hour and minute (as was done in 2.5.x), let's remove the extra space after the ': ' when concatenating the strings.